content/en/examples/service/networking/example-ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: networking.k8s.io/v1
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Hi @RA489 .
Does it make sense to update this example?
Based on:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.20/#ingress-v1-networking-k8s-io
https://kubernetes.io/docs/reference/kubernetes-api/services-resources/ingress-v1/#Ingress

Reply:
Thanks for the review. I think this PR no longer needs to be updated.